tsconfig.json

```diff
@@ -35,6 +35,7 @@
     "exclude": [
         // Type stubs are not part of the top-level project.
         "web/src/types",
+        "help-beta",
```

Comment:
It's important to follow the pattern here of having a comment explaining why this is excluded. By putting it here, you make it look like the "Type stubs ..." comment applies to this, which it does not. I'd suggest something like:

> // This is a separate package with its own TypeScript configuration.

(The whole point of having comments here is to record our decision-making).

Reply:
Fixed!